speechmatics/cli_parser.py

```diff
@@ -283,6 +283,24 @@ def get_arg_parser():
         help="Perform sentiment analysis on the transcript",
     )
 
+    # Parent parser for batch topic-detection argument
+    batch_topic_detection_parser = argparse.ArgumentParser(add_help=False)
+    batch_topic_detection_parser.add_argument(
+        "--topic-detection",
+        dest="topic_detection",
+        action="store_true",
+        default=False,
+        help="Whether to detect topics in transcript",
+    )
+    batch_topic_detection_parser.add_argument(
+        "--topic-detection-topics",
+        dest="topics",
```

Comment:
If the destination var is topics, does the flag need to be --topic-detection-topics or could it just be topics? Is there any possibility we'll have topics as an input to some other thing?

Reply:
argparse has this nice feature that you can also use abbreviations if it isn't ambigiuous, e.g. `--topic science` would work for `--topic-detection-topics`, but yes I agree that the name is a bit weird.
What about `--topics-to-detect`?

Reply:
We can use `--enable-topic-detection` to enable the feature with default topics or `--enable-topic-detection --topics science` as a combination.

Reply:
a couple of different ideas here, can we settle for --detect-topics and --topics XX YY ZZ?

Reply:
I'm happy with that